MobileCenter/MobileCenter/Internals/Utils/MSUtility+Application.m

```diff
@@ -33,13 +33,13 @@ + (void)sharedAppOpenUrl:(NSURL *)url
        completionHandler:(void (^__nullable)(MSOpenURLState state))completion {
   UIApplication *sharedApp = [[self class] sharedApp];
 
-  // FIXME: App extensions does support openURL through NSExtensionContest, we may use this somehow.
+  // FIXME: App extensions don't support openURL through NSExtensionContest, we may use this somehow.
```

Comment:
It does ^^ but yes there is a typo here `NSExtension​Context` instead of `NSExtension​Contest`